tools/ktx/command_create.cpp

```diff
@@ -830,7 +832,7 @@ Create a KTX2 file from various input files.
 */
 class CommandCreate : public Command {
 private:
-    Combine<OptionsCreate, OptionsASTC, OptionsCodec<false>, OptionsMetrics, OptionsCompress, OptionsMultiInSingleOut, OptionsGeneric> options;
+    Combine<OptionsCreate, OptionsASTC, OptionsBasis<false>, OptionsCodecCommon, OptionsMetrics, OptionsCompress, OptionsMultiInSingleOut, OptionsGeneric> options;
```

Comment:
OptionsCompress → OptionsDeflate.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/817/commits/dd12a5ec82f53758c77cbf18c7c2f6e2443a30ff